ui/app/templates/components/search-select.hbs

```diff
@@ -0,0 +1,58 @@
+{{#if shouldUseFallback}}
+  {{component
+    fallbackComponent
+    label=label
+    onChange=(action "onChange")
+    inputValue=inputValue
+    warning=warning
```

Comment:
I think right now the only fallback is `string-list` - does it take a `warning` ?   I think only `kv-object-editor` does because it's teh only one that has any sort of validation on the client. So really I think all of this is to say: we should probably standardize the interface for all of these components and then we could simplify that giant if/else in the form-field template.